packages/sdk/src/__tests__/binding-dx.test.ts

```
@@ -0,0 +1,57 @@
+
+describe("input bindings DX", () => {
+  it("renders path-based json bindings into task prompt", async () => {
+    const root = await mkdtemp(join(tmpdir(), "obora-bind-"));
+    await mkdir(join(root, "artifacts"), { recursive: true });
+    await writeFile(
+      join(root, "artifacts", "submission.json"),
+      JSON.stringify({ score: 1, label: "ok" }, null, 2),
+      "utf8",
+    );
+
+    const workflow = await Workflow.fromYaml(join(process.cwd(), "src/__tests__/fixtures/one-file-binding.yaml"));
```

ESLint

[error] Replace `join(process.cwd(),·"src/__tests__/fixtures/one-file-binding.yaml")` with `⏎······join(process.cwd(),·"src/__tests__/fixtures/one-file-binding.yaml")⏎····`

(prettier/prettier)